### PR TITLE
A few small fixes from Rudolfs' feedback

### PR DESCRIPTION
**Pull request by CodeSandwich**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Find other queries you can run at [Querying the ledger](#querying-the-ledger).
 ## Mining Blocks
 
 You can support the network and receive rewards by mining blocks.
-You can find out how to do that on the [full node setup page](node#Mining_Blocks).
+You can find out how to do that on the [full node setup page](node#mining-Blocks).
```
Comment (NunoAlexandre) [inline docs/getting-started.md]:
```suggestion
You can find out how to do that on the [full node setup page](node#mining-blocks).
```

File: docs/submit-transactions.md [hunk 2]
```diff
@@ -16,13 +16,13 @@ and behaviour of each one of them.
 
 ⓘ All the transactions provide the same [Command-Line Options](#command-line-options).
 
-- Transfer funds from an account to another:
+- Transfer funds from an account to another account, possibly org's:
```
Comment (NunoAlexandre) [inline docs/submit-transactions.md]:
```suggestion
- Transfer funds from the author's account to another account, which can be a local, org's, or user's account.
```

Comment (NunoAlexandre) [reply docs/submit-transactions.md]: A bit too many "accounts' though.

File: docs/submit-transactions.md [hunk 3]
```diff
@@ -16,13 +16,13 @@ and behaviour of each one of them.
...
 
     ``` bash
     radicle-registry-cli account transfer --help
     ```
 
-- Transfer funds from an org's account to another account:
+- Transfer funds from an org's account to another account, possibly another org's:
```
Comment (NunoAlexandre) [inline docs/submit-transactions.md]:
```suggestion
Transfer funds from an org's account to another account, which can be a local, org's, or user's account.
```

File: docs/submit-transactions.md [hunk 4]
```diff
@@ -111,8 +111,7 @@ can either be specified as options at the end of a command or as environment var
 
     Mandatory for all transactions. The author option lets you specify the account that is
     used to sign the transaction. You can use the `account` command group to manage your
-    accounts.
-    that you want to use to sign a transaction.
+    accounts, that you want to use to sign a transaction.
```
Comment (NunoAlexandre) [inline docs/submit-transactions.md]: This is unclear if it's referring to managing the accounts (i.e, if you want to sign a transaction, you need to manage your accounts), or to the accounts themselves. I suggest making this block:

 > Mandatory for all transactions. The author option lets you specify the local account that 
> is used to sign the transaction. You can use the `account` command group to generate 
> and list the local accounts that you can to use to sign transactions.

**Review by NunoAlexandre — CHANGES_REQUESTED** [review]
Some important bits to correct.

File: docs/submit-transactions.md (hunk 4 above)
Comment (CodeSandwich) [reply docs/submit-transactions.md]: I've just replaced the dot with the comma D: Your proposition is fine though :+1:

Comment (CodeSandwich) [reply docs/submit-transactions.md]: Fixed

File: docs/submit-transactions.md (hunk 3 above)
Comment (CodeSandwich) [reply docs/submit-transactions.md]: Fixed

File: docs/submit-transactions.md (hunk 2 above)
Comment (CodeSandwich) [reply docs/submit-transactions.md]: That's the only word we have :smile: Fixed.

File: docs/getting-started.md (hunk 1 above)
Comment (CodeSandwich) [reply docs/getting-started.md]: Fixed

**Review by NunoAlexandre — APPROVED** [review]
:guitar:

**Pull request merged**